src/security/SafeCommandExecutor.ts

```
@@ -0,0 +1,258 @@
+    if (!/^[a-zA-Z0-9_-]+$/.test(command)) {
+      throw new Error(`Command contains invalid characters: ${command}`);
+    }
+
+    return command;
+  }
+
+  private static sanitizeArgument(arg: string): string {
+    // Remove dangerous characters and patterns
+    let sanitized = arg;
+    
+    for (const pattern of this.DANGEROUS_PATTERNS) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)